hms-api/src/handler/comment/create.ts

```diff
@@ -0,0 +1,19 @@
+import {buildResponse} from '../../rest/responses';
+import {createComment} from '../../service/idea_comment-service';
+import {wrapHandler} from '../handler-wrapper';
+import CommentCreateRequest from '../../rest/Comment/CommentCreateRequest';
+import CommentCreateResponse from '../../rest/Comment/CommentCreateResponse';
+
+export async function create(event, context, callback) {
+  await wrapHandler(async () => {
+    const request = CommentCreateRequest.parse(event.body);
+    const comment = await createComment(
+      request.ideaId,
```

Comment:
in the URL you have the ID of the idea as part of the path, but here you also have it as part of the request

Reply:
sorry I understand what you mean here